python/ccf/clients.py

```diff
@@ -26,13 +26,28 @@
 import ccf.commit
 
 
-def truncate(string: str, max_len: int = 256):
+def truncate(string: str, max_len: int = 128):
     if len(string) > max_len:
         return f"{string[: max_len]} + {len(string) - max_len} chars"
     else:
         return string
 
 
+def info_or_capture(line, log_capture=None, depth=0):
+    if log_capture is None:
+        LOG.opt(colors=True, depth=depth + 1).info(line)
+    else:
+        log_capture.append(line)
+
+
+def flush_info(lines, log_capture=None, depth=0):
```

Comment:
This (and the call above) needs to be split to avoid a deps cycle, but please ignore that for now.